microservices/nris_api/backend/app/routes.py

```diff
@@ -0,0 +1,28 @@
+from app.utils.logger import get_logger
```

Comment:
This file is proposing a different route-management pattern than what is used in CORE at the moment? Is there a reason for this, or this is just filler code while the deployment is being set up initially?

Reply:
Not sure it's a different route management, can you please clarify. We have to access the application logger through different means if it's the scheduler running the app or a request... so i was hoping to abstract that complexity so it's easier to get logging in from the start.

Reply:
are you referring to `backend/app/nris` being equal to `python-backend/app/api`, but it's `backend/app/utils` instead of `python-backend/app/api/utils`, if so that's a good eye, but there are multiple branches based on these changes, so i'd have to ask the team for a full stop when i make that change and merge, so I agree utils should be under `backend/app/nris`, but i don't think that change can be made right now.

Reply:
I may just be misunderstanding the purpose of this file 😅 

By looking at it, I thought this would be replacing an `/api/model/namespace/model.py` file by putting all of the routes for this microservice into one `routes` file.

If that's not correct then just disregard my comment. If it is correct, my question is the benefit proposition of having this different pattern in the microservice vs the cognitive overhead of differing patterns.

Reply:
I see where the confusion lies, in the core backend we currently have a function called register_routes, it's getting quite long with all the namespace imports and exception handlers, so i made a file called routes just holds that domain of initialization. 

Does that make sense. I am not suggesting we have a folder that contains all of the resource files.

Reply:
Yep. All good 👍 I'll mark this as resolved